### PR TITLE
Fix copypasta error in bugdocs

### LINKED ISSUE #997: Copy+paste error in bug docs
https://github.com/pret/pokecrystal/blob/master/docs/bugs_and_glitches.md#checkownmonanywhere-does-not-check-the-day-care

### PR DESCRIPTION
**Pull request by coltongit**
Fixes #997

### CODE REVIEW

File: docs/bugs_and_glitches.md [hunk 1]
```diff
@@ -2439,7 +2439,7 @@ This bug can prevent you from talking to Eusine in Celadon City or encountering
 ```diff
  	; If there are no monsters in the party,
  	; the player must not own any yet.
--; BUG: CheckOwnMon does not check the Day-Care (see docs/bugs_and_glitches.md)
```
Comment (Rangi42) [inline docs/bugs_and_glitches.md:2442]: Just remove everything in the diff prior to the actual `BUG:` comment.

Comment (coltongit) [reply docs/bugs_and_glitches.md:2442]: Is replacing that with ... OK too?

File: docs/bugs_and_glitches.md [hunk 2]
```diff
@@ -2437,13 +2437,7 @@ This bug can prevent you from talking to Eusine in Celadon City or encountering
...
 
 ```diff
- 	; If there are no monsters in the party,
- 	; the player must not own any yet.
--; BUG: CheckOwnMon does not check the Day-Care (see docs/bugs_and_glitches.md)
- 	ld a, [wPartyCount]
- 	and a
- 	ret z
-
+...
```
Comment (Rangi42) [inline docs/bugs_and_glitches.md:2440]: Why?
```suggestion
...
```

Comment (coltongit) [reply docs/bugs_and_glitches.md:2440]: I don't have that character/I'm not sure how to type it. Thanks

**Pull request closed without merging**